backend/src/migrations/20230405072520_add_images.ts

```diff
@@ -0,0 +1,62 @@
+import { type Knex } from 'knex';
+
+const uuid = 'uuid_generate_v4()';
+
+enum Gender {
+    Male = 'male',
+    Female = 'female',
+}
+
+const TABLE_NAME = 'user_profile';
+const FOREIGN_TABLE_NAME = 'users';
+
+const ColumnName = {
+    ID: 'id',
+    USER_ID: 'userId',
+    FIRST_NAME: 'firstName',
+    LAST_NAME: 'lastName',
+    SEX: 'sex',
+    DATE_OF_BIRTH: 'dateOfBirth',
+    LANGUAGE: 'language',
+    CURRENCY: 'currency',
+    AVATAR_URL: 'avatarUrl',
+    CREATED_AT: 'createdAt',
+    UPDATED_AT: 'updatedAt',
+};
+const RelationRule = {
+    CASCADE: 'CASCADE',
+    SET_NULL: 'SET NULL',
+};
+async function up(knex: Knex): Promise<void> {
+    await knex.schema.dropTableIfExists(TABLE_NAME);
```

Comment:
You don't need to drop table, you can edit the existing one instead

Reply:
Resolved in [6ccc578](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/314/commits/6ccc578e7713a42dbb64accaa2bbbecc31440000)